gensim/models/ldaseqmodel.py

```diff
@@ -578,8 +575,7 @@ def doc_topics(self, doc_number):
             Probability for each topic in the mixture (essentially a point in the `self.num_topics - 1` simplex.
 
         """
-        doc_topic = np.copy(self.gammas)
-        doc_topic /= doc_topic.sum(axis=1)[:, np.newaxis]
+        doc_topic = self.gammas / self.gammas.sum(axis=1)[:, np.newaxis]
         return doc_topic[doc_number]
```

Comment:
why not to add copy here?

Reply:
`numpy` returns new array as a result of division operation. However operation '/=' does not create new arrays.

Reply:
I mean why we don't copy `doc_topic[doc_number]` (to avoid mutation from external code)?